crates/matrix-sdk/src/widget_api/messages/to_widget.rs

```diff
@@ -0,0 +1,48 @@
+use serde::{Deserialize, Serialize};
+
+use crate::widget_api::messages::{MessageBody, OpenIdState, MatrixEvent};
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "action")]
+pub enum ToWidgetMessage {
+    #[serde(rename = "capabilities")]
+    SendMeCapabilities(MessageBody<(), SendMeCapabilitiesResponse>),
+    #[serde(rename = "notify_capabilities")]
+    CapabilitiesUpdated(MessageBody<CapabilitiesUpdatedRequest, ()>),
+    #[serde(rename = "openid_credentials")]
+    OpenIdCredentials(MessageBody<OpenIdCredentialsRequest, ()>),
+    #[serde(rename = "sent_to_device")]
+    SendToDevice(MessageBody<SendToDeviceRequest, ()>),
+    #[serde(rename = "send_event")]
+    SendEvent(MessageBody<MatrixEvent, ()>)
+}
```

Comment:
It generally looks good, but it has a problem similar to `FromWidgetMessage`, i.e. it lacks responses and it allows requests to have a response field set.

Should we probably replace `MessageBody<>` with separate `RequestBody` and `ResponseBody` if that's easier to understand? (I believe that was something that confused you 🙂)